app/javascript/packages/spinner-button/spinner-button-element.ts

```diff
@@ -42,7 +42,7 @@ export class SpinnerButtonElement extends HTMLElement {
   }
 
   get button(): HTMLElement {
-    return this.querySelector('a,button:not([type]),[type="submit"],[type="button"]')!;
+    return this.querySelector('.usa-button')!;
```

Comment:
I suspect this was premature optimization on past Andrew's part. I don't know that we ever render any spinner buttons that aren't `.usa-button`.